docs/_docs/localization.md

```diff
@@ -10,10 +10,17 @@ We are developing tools for developers to localize the figma design into differe
 The first introduced tool is a figma plugin that generates a strings xml file containing the strings
 used by the figma design.
 
-## 1. Build the localization plugin
+## 1. Run the localization plugin
 
-The localization plugin is still under development and is only available from building the source code.
-Follow [Get Started][1] page to download the source code.
+We have published our first version of the localization plugin. You can run it through:
+**Figma Main Menu -> Plugins -> From Google**
+   **-> Automotive Design For Compose -> Localization Tool -> Generate String Resource**
```

Comment:
I don't think it's under the "From Google" section -- at least it's not for me.

Reply:
There are two lists sorted alphabetically under "From Google". It is in the bottom.

Reply:
Oh, never noticed that, ok.